src/lib/viewers/doc/_docBase.scss

```diff
@@ -295,6 +295,17 @@ $thumbnail-sidebar-width: 191px; // Extra pixel to account for sidebar border
         }
     }
 
+    //overwriting background image in pdf_viewer.css. Was causing 404's
+    /* stylelint-disable declaration-no-important */
+    .pdfViewer .page.loadingIcon {
+        background-image: none !important;
+    }
+
+    .pdfViewer .page.loadingIcon::after {
```

Comment:
We have other `.page .loadingIcon` styles a few lines up. Can we move these rules into the nested rules?

Reply:
done